test_signature_coherence_CLI.py

```diff
@@ -0,0 +1,42 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# CLI INTERFACE THAT EVALUATES THE SIMILARITY BETWEEN STACK TRACES IN A GIVEN SIGNATURE.
+
+
+import download_data
+import crash_similarity
+import argparse
+import utils
+from datetime import timedelta
+
+parser = argparse.ArgumentParser(description = 'Test similarities between two signatures')
```

Comment:
>  Don't use spaces around the = sign when used to indicate a keyword argument or a default parameter value. 

https://www.python.org/dev/peps/pep-0008/

the same in `crash_similarity.py` and in `test_top_similar_traces_CLI.py`